pkg/synopsysctl/cmd_destroy.go

```diff
@@ -65,7 +66,11 @@ var destroyCmd = &cobra.Command{
 				}
 				if metav1.NamespaceAll != namespace {
 					operatorNamespace = namespace
+				} else {
+					log.Infof("destroy Synopsys Operator defaulting to namespace '%s'...", operatorNamespace)
```

Comment:
These else conditions aren't needed.  Just put a log before the destroy call saying "destroying Synopsys Operator in NS ..." or whatever.

Reply:
Just caution the user that we are defaulting to synopsys operator

Reply:
We can accomplish that same information with a common log message before destroy.  Whether we default or not the user wouldn't be able to do anything with the fact that we are using defaults, so might as well simplify the code and just have a single log message denoting where we are destroying the operator.